include/alpaka/kernel/KernelBundle.hpp

```diff
@@ -32,7 +32,6 @@ namespace alpaka
         {
         }
 
-    private:
```

Comment:
Why no longer `private` ?

Reply:
We access the member from outside. Writing getter would be possible but IMO not strictly required